app/states/unified_database.py

```
@@ -0,0 +1,1208 @@
+        Update a person, saving a version snapshot before updating.
+        """
+        with self.get_session() as session:
+            person = session.get(UnifiedPerson, person_id)
+            if not person:
+                return None
+            version_count = session.exec(select(UnifiedPersonVersion).where(UnifiedPersonVersion.person_id == person.id)).count()
+            version = UnifiedPersonVersion(
+                person_id=person.id,
+                version_number=version_count + 1,
+                data=json.dumps({k: getattr(person, k) for k in person.__fields__.keys()}),
+                changed_at=datetime.utcnow(),
```

Ruff (0.14.7)

`datetime.datetime.utcnow()` used

(DTZ003)